mavros_msgs/msg/Debug.msg

```diff
@@ -0,0 +1,26 @@
+# Msg for Debug MAVLink API
+#
+# Supported types:
+# DEBUG			http://mavlink.org/messages/common#DEBUG
+# DEBUG_VECTOR			http://mavlink.org/messages/common#DEBUG_VECT
+# NAMED_VALUE_FLOAT		http://mavlink.org/messages/common#NAMED_VALUE_FLOAT
+# NAMED_VALUE_INT		http://mavlink.org/messages/common#NAMED_VALUE_INT
+# @TODO: add support for DEBUG_ARRAY (https://github.com/mavlink/mavlink/pull/734)
+
+# debug types for msg
+uint8 DEBUG_TYPE		= 0
+uint8 DEBUG_VECT_TYPE		= 1
+uint8 NAMED_VALUE_FLOAT_TYPE	= 2
+uint8 NAMED_VALUE_INT_TYPE	= 3
+
+std_msgs/Header header
+
+uint8 type
+
+uint8 index			# index value of DEBUG value
+
+string name			# value name/key
+
+float32 value_float		# float value for NAMED_VALUE_FLOAT and DEBUG
+int32 value_int		# int value for NAMED_VALUE_INT
+geometry_msgs/Vector3 vector	# DEBUG vector
```

Comment:
Add variable sized array for debug array. But always publish all fields, even if they zero. We cant know real size of values array.

Reply:
we could do that but currently just `DEBUG_VECT` is supported